docs/how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -107,11 +109,9 @@ for passing annotations to Kata from containerd:
 $ cat /etc/containerd/config
 ....
 
-[plugins.cri.containerd.runtimes.kata]
-           runtime_type = "io.containerd.runc.v1"
-           pod_annotations = ["io.katacontainers.*"]
-           [plugins.cri.containerd.runtimes.kata.options]
-             BinaryName = "/usr/bin/kata-runtime"
+         [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
+           runtime_type = "io.containerd.kata.v2"
+           pod_annotations = ["io.kata-containers.*"]
```

Comment:
@liubin, shouldn't it be `"io.katacontainers.*"` instead of `"io.kata-containers.*"`?

Reply:
@fidencio , you are right.